CHANGELOG.md

```diff
@@ -2,6 +2,8 @@ UNRELEASED
 ----------
 
 * [ [#951](https://github.com/digitalfabrik/integreat-cms/issues/951) ] Add possibility to create categories for POIs
+* [ [#1742](https://github.com/digitalfabrik/integreat-cms/issues/1742) ] Add last modified date to media sidebar
+
```

Comment:
```suggestion
```
Unfortunately, the changelog needs exactly one new line after the unreleased headline and exactly two new lines between the bullet points and the next version headline... :grimacing: